docs/README.md

```diff
@@ -1042,6 +1045,49 @@ The resulting staging repository will be available in the output named `staging-
           nexus-password: "${{ secrets.NEXUS_PASSWORD }}"
 ```
 
+### nexus-create-tag
+
+Creates a tag on Nexus with a specific name. The resulting tag will be available in the output named `tag-name`.
+
+```yaml
+      - uses: Alfresco/alfresco-build-tools/.github/actions/nexus-create-tag@ref
+        with:
+          tag-name:  ${{ needs.load-release-info.outputs.version }}
+          nexus-username: ${{ secrets.NEXUS_USERNAME }}
+          nexus-password: ${{ secrets.NEXUS_PASSWORD }}
+```
+
+### nexus-associate-tag
+
+Associates tag to artifacts on Nexus. The artifacts are filtered by provided maven group-id and version and then associated with the provided tag.
+
+```yaml
+      - uses: Alfresco/alfresco-build-tools/.github/actions/nexus-associate-tags@ref
+        with:
+          tag-name: ${{needs.load-release-info.outputs.version }}
```

Comment:
provide an example value, maybe the same used in nexus-create-tag example if it's the same